xds/internal/xdsclient/authority.go

```diff
@@ -327,9 +327,9 @@ func decodeAllResources(opts *xdsresource.DecodeOptions, rType xdsresource.Type,
 		return ret, md, nil
 	}
 
-	typeStr := rType.TypeEnum().String()
+	typeName := rType.TypeName()
 	md.Status = xdsresource.ServiceStatusNACKed
-	errRet := combineErrors(typeStr, topLevelErrors, perResourceErrors)
+	errRet := combineErrors(typeName, topLevelErrors, perResourceErrors)
```

Comment:
Nit: inline `rType.TypeName()` here and get rid of `typeName` variable.

Reply:
Done